### DISCUSSION TITLE
How can i know the current iter?

### DISCUSSION THREAD

**Discussion by morestart** [Lightning-AI/pytorch-lightning#11215] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
i want use iter to control multi_scale

**Comment by morestart** [comment]
ok i know the answer

**Comment by akihironitta** [comment, marked as answer]
Just for anyone looking at this Discussion, here's the list of relevant properties:
- https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html#global-step
- https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html#current-epoch
